vm_supervisor/network/interfaces.py

```diff
@@ -40,12 +58,25 @@ async def create(self):
                 self.device_name,
             ]
         )
+        ipv6_gateway = self.host_ipv6
+        run(
+            [
+                ip_command,
+                "addr",
+                "add",
+                str(ipv6_gateway),
+                "dev",
+                self.device_name,
+            ]
+        )
         run([ip_command, "link", "set", self.device_name, "up"])
+        self.ndp_proxy.add_range(self.device_name, ipv6_gateway.network)
         logger.debug(f"Network interface created: {self.device_name}")
 
     async def delete(self) -> None:
         """Asks the firewall to teardown any rules for the VM with id provided.
         Then removes the interface from the host."""
         logger.debug(f"Removing interface {self.device_name}")
         await asyncio.sleep(0.1)  # Avoids Device/Resource busy bug
+        self.ndp_proxy.delete_range(self.device_name)
```

Comment:
Should this go before the `asyncio.sleep` ?

Reply:
I was wondering what this sleep fixes? At first glance it just seems like this occurs because we don't down the device before deleting it.